doc: TF-M updates for the NCS 2.0.0 release (#7614)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

* Updated:

>   * :ref:`ug_nrf5340` and :ref:`ug_nrf91` with information about Trusted Firmware-M replacing Secure Partition Manager in being the default solution for creating a Trusted Execution Environment.

Reviewer: ```suggestion
  * :ref:`ug_nrf5340` and :ref:`ug_nrf91` with information about Trusted Firmware-M replacing Secure Partition Manager as the default solution for creating a Trusted Execution Environment.
```
